### ISSUE TITLE
[rust client] Generated README auth section is for Go, not Rust

### ISSUE THREAD

**Issue by mandrean** [OpenAPITools/openapi-generator#1488]
##### Description

When generating a Rust client from a given API spec that uses authentication (e.g. Basic Auth or API Key), the auth section in the generated README contains a code example for Go, not rust:

![image](https://user-images.githubusercontent.com/4631744/48675070-76571980-eb54-11e8-873c-af3fbe0854f9.png)

##### openapi-generator version

Tried both the latest stable release, and the master branch @ 7f8ff35245.

##### OpenAPI declaration file content or url

Any OAS spec will do as the error lies from a copy-paste error in the mustache templates: https://github.com/OpenAPITools/openapi-generator/blob/7f8ff35245edc4f31354068efb9d26e2b9286a83/modules/openapi-generator/src/main/resources/rust/README.mustache

##### Command line used for generation

```
./run-in-docker.sh generate -i spec.yaml -g rust -o /gen/out
```

##### Steps to reproduce

Generate a Rust client from an OAS 3 spec. Check the `README.md`.

##### Suggest a fix/enhancement

Update the mustache templates with correct Rust code instead of Go: https://github.com/OpenAPITools/openapi-generator/blob/7f8ff35245edc4f31354068efb9d26e2b9286a83/modules/openapi-generator/src/main/resources/rust/README.mustache

**Comment by wing328** [comment]
@mandrean thanks for reporting the issue. May I know if you've time to contribute a fix to the auto-generated doc?

cc @frol (2017/07) @farcaller (2017/08) @bjgill (2017/12)

**Comment by mandrean** [comment]
@wing328, yeah sure! I’ll look into it.

**Comment by FinnRG** [comment]
This issue was resolved in 3df525ee339a368ec59d945447e5677f004a8619.
